pcsx2-qt/Settings/GraphicsSettingsWidget.ui

```diff
@@ -104,6 +104,11 @@
            <string>Widescreen (16:9)</string>
           </property>
          </item>
+         <item>
```

Comment:
Indenting is not correct

Reply:
I don't understand this, could you help me correct it?

Reply:
Sorry, that indent is correct.